apps/site/next-data/providers/blogData.ts

```diff
@@ -1,12 +1,19 @@
 import { cache } from 'react';
 
-import generateBlogData from '@/next-data/generators/blogData.mjs';
 import { BLOG_POSTS_PER_PAGE } from '@/next.constants.mjs';
+import { blogData } from '@/next.json.mjs';
 import type { BlogCategory, BlogPostsRSC } from '@/types';
 
-const { categories, posts } = await generateBlogData();
+function getPosts() {
+  return blogData.posts.map(post => ({
+    ...post,
+    date: new Date(post.date),
```

Comment:
Actually, I think this should just be:

```js
const blogPosts = cache(() => blogData.posts.map(...));
```

Reply:
ah ok! sorry for the silly mistake :bow: 

Updated :slightly_smiling_face::+1: